src/osd/scrubber/scrub_reservations.h

```diff
@@ -93,15 +117,19 @@ class ReplicaReservations {
   bool handle_reserve_grant(OpRequestRef op, pg_shard_t from);
 
   /**
+   * React to an incoming reservation rejection.
+   *
    * Verify that the sender of the received rejection is the replica we
-   * were expecting a reply from.
-   * If this is so - just mark the fact that the specific peer need not
-   * be released.
+   * were expecting a reply from, and that the message isn't stale.
+   * If a real rejection: log it, and mark the fact that the specific peer
+   * need not be released.
```

Comment:
"real rejection" and "stale" aren't useful unless you already know how this mechanism works.  Refer the reader to comments explaining the mechanisms.